src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -526,7 +526,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
         Private Function ParseExecutableStatementCore() As StatementSyntax
             Dim outerContext As New CompilationUnitContext(Me)
-            Dim fakeBegin = SyntaxFactory.SubStatement(Nothing, Nothing, InternalSyntaxFactory.MissingKeyword(SyntaxKind.SubKeyword),
+            Dim builder As New CoreInternalSyntax.SyntaxListBuilder(1)
+            builder.Add(InternalSyntaxFactory.Token(Nothing, SyntaxKind.AsyncKeyword, Nothing))
```

Comment:
>builder.Add(InternalSyntaxFactory.Token(Nothing, SyntaxKind.AsyncKeyword, Nothing)) [](http://example.com/codeflow?start=12&length=83)

This change doesn't look justified. Now it breaks parsing for scenarios that aren't in async context.